### ISSUE TITLE
Weird runtime with photos

### ISSUE THREAD

**Issue by Alek2ander** [tgstation/tgstation#5673]
```
runtime error: bad resource file
proc name: show (/obj/item/weapon/photo/proc/show)
source file: photography.dm,60
usr: Nickolas Mason (/mob/living/carbon/human)
src: the photo (/obj/item/weapon/photo)
call stack:
the photo (/obj/item/weapon/photo): show(Nickolas Mason (/mob/living/carbon/human))
the photo (/obj/item/weapon/photo): examine(Nickolas Mason (/mob/living/carbon/human))
Nickolas Mason (/mob/living/carbon/human): Examine(the photo (/obj/item/weapon/photo))
the photo (/obj/item/weapon/photo): attack self(Nickolas Mason (/mob/living/carbon/human))
Nickolas Mason (/mob/living/carbon/human): ClickOn(the photo (/obj/item/weapon/photo), "icon-x=16;icon-y=18;left=1;scr...")
the photo (/obj/item/weapon/photo): Click(null, "mapwindow.map", "icon-x=16;icon-y=18;left=1;scr...")
```

Line 60:
`user << browse_rsc(img, "tmp_photo.png")`

**Comment by Firecage** [comment]
Huh, I am unable to reproduce this bug on me end.

**Comment by tedward1337** [comment]
Most likely fixed by #8658

**Comment by phil235** [comment]
good enough for me
